src/clickhouse_migrations/cmd.py

```diff
@@ -91,6 +91,12 @@ def get_context(args):
         type=bool,
         help="Dry run mode",
     )
+    parser.add_argument(
+        "--secure",
+        default=os.environ.get("SECURE", "1"),
```

Comment:
@dimoschi it's better to leave backward compatible with previous versions, 
so, can you put default value as zero?
it's also will fix tests